### PR TITLE
PIM-5950: Render string filter on value update to toggle input

### PR DESCRIPTION
**Pull request by grena**
| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | N
| Added Behats                      | 
| Changelog updated                 | N
| Review and 2 GTM                  |
| Micro Demo to the PO (Story only) |
| Migration script                  | N
| Tech Doc                          | N

### CODE REVIEW

**Comment by jjanvier** [comment]
where is Jasmine?

**Comment by jjanvier** [comment]
gtm

**Comment by juliensnz** [comment]
Add behat? GTM

**Comment by juliensnz** [comment]
GTM, good job dude

File: features/Pim/Behat/Decorator/Export/Filter/StringDecorator.php [hunk 1]
```diff
@@ -41,4 +42,22 @@ public function filter($operator, $value)
             );
         }
     }
+
+    /**
+     * Return whether this filter input value is visible
+     *
```
Comment (damien-carcel) [inline features/Pim/Behat/Decorator/Export/Filter/StringDecorator.php]: `@throws`

**Comment by damien-carcel** [comment]
One PHPDoc comment, GTM

File: features/Pim/Behat/Decorator/Export/Filter/StringDecorator.php (hunk 1 above)
Comment (grena) [inline features/Pim/Behat/Decorator/Export/Filter/StringDecorator.php]: it throws nothing

**Pull request merged**